guidelines/index.html

```diff
@@ -1433,6 +1441,10 @@
 					<p>translation of one language, generally a spoken language, into a <a>sign language</a></p>
 					<p>True sign languages are independent languages that are unrelated to the spoken language(s) of the same country or region.</p>
 				</dd>
+        <dt><dfn>Significant animation</dfn></dt>
+				<dd>
+					<p>animation which continues for more than 3 seconds, or is synchronized with a user action, and affects more than 1/3 of the viewport.</p>
```

Comment:
as noted above, "significant" needs to be explained/emphasised here. what is "significant" may be subjective, but giving examples would help...